app/models/concerns/background_deletion.rb

```diff
@@ -5,32 +5,17 @@ module BackgroundDeletion
 
   included do
     class_attribute :background_deletion, default: [], instance_writer: false
+    class_attribute :background_deletion_method, default: :destroy!, instance_writer: false
+    class_attribute :background_deletion_scope_name, default: :all, instance_writer: false
```

Comment:
Do we really need this? Seems that the only place it is currently used in is the alerts, but it's commented out...

Reply:
If you say, I will remove. I wanted in the PoC to show the two approaches. But it seems we are on the same page that it was an overlook originally so we just remove any objects, not change state.